lib/pyfrc/physics/drivetrains.py

```diff
@@ -1,3 +1,4 @@
+import math
```

Comment:
Hmm. GitHub seems to want to keep showing this outdated diff.

Anyway, good thing you've fixed the issue here.